tests/e2e-playwright/README.md

```diff
@@ -0,0 +1,145 @@
+# Sensei e2e tests
+
+## Quickstart
+You should first make sure that the [dependencies are installed](#dependencies). 
+Run `npm run test:e2e`.
+
+## Running
+Run `npm run test:e2e` headless (browser in background) mode.
+Run `npm run test:e2e:debug` headed mode (browser in foreground).
+
+
+## Dependencies
+Before running the tests, you should make sure that the following dependencies are installed:
+* Docker. For Mac you can install you can use [Colima](https://github.com/abiosoft/colima) 
+* [Playwright]. (https://playwright.dev/docs/intro#manually) 
```

Comment:
Is it possible to install Playwright automatically with other Node packages?
I'm not sure how it was installed for me. I only had to run `npx playwright install` (I got the corresponding warning when running e2e-tests).

Reply:
@merkushin  I thought the same, but I saw `npx playwright install` check the playwright config and install the expected browser, they are doing it separately to avoid slowdown common npm install usage, avoiding downloading it when it is not necessary.